conda/environments/cudf_dev_cuda11.5.yml

```diff
@@ -7,6 +7,7 @@ channels:
   - rapidsai-nightly
   - dask/label/dev
   - conda-forge
+  - gmarkall
```

Comment:
You will need to have the channel added into `build.sh` files in the similar way I did in my PR here: https://github.com/rapidsai/cudf/pull/11617/files

Reply:
Thanks, I've just seen this whilst I was making a change with a similar aim. I will have a look at the way you did it too.